test/scenario-migration-generate/src/Evergreen/Migrate/V2.elm

```diff
@@ -1,4 +1,4 @@
-module Evergreen.Migrate.V2 exposing (backendModel, backendMsg, frontendModel, frontendMsg, toBackend, toFrontend)
+module Evergreen.Migrate.V2 exposing (..)
```

Comment:
The migration generation should be explicitly exposing `backendModel, backendMsg, frontendModel, frontendMsg, toBackend, toFrontend`. Not sure if we should revert that just so the test passes?

@supermario Thoughts?